test/CMakeLists.txt

```diff
@@ -10,5 +10,7 @@ target_link_libraries(test_util ${TERRIER_LINK_LIBS})
 add_dependencies(test_util gtest)
 
 add_subdirectory(common)
+add_subdirectory(expression)
```

Comment:
In theory since expression is a subfolder of type, the tests should be under type as well.